app/services/permitted_attributes/checkout.rb

```diff
@@ -11,24 +13,13 @@ def call
           :existing_card_id, :shipping_method_id,
           payments_attributes: [
             :payment_method_id,
-            source_attributes: payment_source_attributes
+            source_attributes: PermittedAttributes::PaymentSource.attributes
```

Comment:
since this `PaymentSource` belongs to `PermittedAttributes` module Ruby's constant lookup should find it without having to specify `PermittedAttributes::`. Have you tried it?

Reply:
I havent. I think this is better than using the implicit namespace. Also because in this case the namespace is informative: it's not a payment source, it's the permitted attributes of a payment source.

Reply:
then the issue is with the actual class name IMO but fair enough :+1:

Reply:
thanks!
Can you review the PR? This PR is a release blocker I think.